lib/bundles/inspec-compliance/cli.rb

```diff
@@ -179,7 +179,7 @@ def upload(path) # rubocop:disable Metrics/MethodLength, Metrics/AbcSize, Percei
       end
 
       # determine user information
-      if config['token'].nil? || config['user'].nil?
+      if (config['token'].nil? && config['refresh_token'].nil?) || config['user'].nil?
```

Comment:
Having one or the other is now a valid use case, if a token is still in the config it will default to using that (which is the legacy behavior